packages/atomic/src/components/result-template-components-v1/atomic-result-multi-text/atomic-result-multi-text.tsx

```diff
@@ -0,0 +1,164 @@
+import {Component, Prop, h, State, VNode} from '@stencil/core';
+import {
+  BreadcrumbManager,
+  buildBreadcrumbManager,
+  Result,
+  ResultTemplatesHelpers,
+} from '@coveo/headless';
+import {ResultContext} from '../../result-template-components/result-template-decorators';
+import {getFieldValueCaption} from '../../../utils/field-utils';
+import {
+  Bindings,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {titleToKebab} from '../../../utils/utils';
+
+const listItemClasses = 'inline-block';
+
+/**
+ * The `atomic-result-multi-text` component renders the values of a multi-value string result field.
+ * @part result-multi-text-separator - The separator between two field values.
+ * @part result-multi-text-value - A field value.
+ * @slot result-multi-text-value-* - Allows the personalization of a particular field value based on its indexed name.
+ */
+@Component({
+  tag: 'atomic-result-multi-text-v1',
+  styleUrl: 'atomic-result-multi-text.pcss',
+  shadow: true,
+})
+export class AtomicResultMultiText implements InitializableComponent {
+  public breadcrumbManager!: BreadcrumbManager;
+
+  @InitializeBindings() public bindings!: Bindings;
+  @ResultContext() private result!: Result;
+
+  @State() public error!: Error;
+
+  /**
+   * The result field which the component should use.
+   * This will look in the Result object first, and then in the Result.raw object for the fields.
+   * It is important to include the necessary field in the ResultList component.
+   */
+  @Prop() public field!: string;
+
+  /**
+   * If present, will attempt to split the field into multiple values using wherever this character is encountered.
+   */
+  @Prop() public delimitingCharacter?: string;
```

Comment:
Is there a default we could set here? or does the character change a lot across clients?

Reply:
Honestly, looking back at it, I'll remove this prop since it looks like properly indexed results should return an array. If somebody requests this feature, we can add it back.